install_config/configuring_vsphere.adoc

```diff
@@ -145,6 +145,13 @@ from the vCenter Inventory.
 Configuring {product-title} for VMWare vSphere requires the
 *_/etc/origin/cloudprovider/vsphere.conf_* file, on each node host.
 
+[IMPORTANT]
```

Comment:
I think this information might be helpful in the upgrade docs, too.

Reply:
I checked the '_Upgrading Clusters_' section and it does not talk about `vsphere` at all.